electricpy/__init__.py

```diff
@@ -4269,28 +4426,61 @@ def secondary(val, Np, Ns=1,invert=False):
                 used when reflecting current across a
                 voltage transformer, or voltage across a
                 current transformer.
-    
+
     Returns
     -------
     reflection: complex
                 The reflected value referred to the secondary
                 side according to Np and Ns.
     """
     if invert:
-        return( val * Np/Ns )
-    return( val * Ns/Np )
+        return (val * Np / Ns)
+    return (val * Ns / Np)
+
+
+def tap_changing_transformer(v1, v2, p, q, r, x):
+    '''
+        calulating turn ration for on load tap chaining transformer
+        
+        Parameters
+        ----------
+        
+        v1: generating station voltage
+        
+        v2: distribution network voltage
```

Comment:
How about using `Vdist` or `Vnetwork` for this variable to make it more descriptive?

Reply:
updated in subsequent PR